core/src/main/java/com/linecorp/armeria/server/SetPathMapping.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2015 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server;
+
+import java.util.Set;
+
+import com.google.common.collect.ImmutableSet;
+
+public final class SetPathMapping extends AbstractPathMapping {
```

Comment:
How about:

- Renaming this class to `AnyOfPathMapping`
- Accepting `Iterable<PathMapping>` as a constructor parameter rather than just doing exact matching
- Adding `PathMapping.anyOf(PathMapping...)` and `PathMapping.anyOf(Iterable<? extends PathMapping>)`
- Making this class package-local?